server/controllers/list.js

```diff
@@ -0,0 +1,73 @@
+const expressAsyncHandler = require("express-async-handler");
```

Comment:
```suggestion
const asyncHandler = require("express-async-handler");
```
Small thing but just want to match the rest of the controllers